modules/entity_version_history/entity_version_history.module

```diff
@@ -33,3 +35,13 @@ function entity_version_history_entity_type_alter(array &$entity_types) {
     $entity_types[$entity_type]->setLinkTemplate('drupal:entity-version-history', $history_path);
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ */
+function entity_version_history_entity_version_history_settings_insert(EntityInterface $entity) {
```

Comment:
It is your own entity type so you dont need to implement this hook. You can do it either in the Entity class or better yet in your own storage handler for this entity type (postSave method)

Reply:
I explained two choices here, clearly stating which one is better. True, the better one would have taken 10 more minutes to implement. 

Also what is the first thing you need to look at when overriding a class method?